### PR TITLE
build: card & menu components

### PR DESCRIPTION
**Pull request by AyaQunoo**

### CODE REVIEW

File: client/src/components/DropDownMenu/menuItems.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  'English',
+  'Spanish',
+  'French',
+  'German',
+];
+
+const menuItems: MenuProps['items'] = languages.map((language) => ({
+  label: language,
+  key: language,
+  icon:' ll'
```
Comment (devstm) [inline client/src/components/DropDownMenu/menuItems.ts]: image

Comment (devstm) [reply client/src/components/DropDownMenu/menuItems.ts]: icon ll !!!!!!!!!!!!!!1

**Review by devstm — CHANGES_REQUESTED** [review]

File: client/src/pages/Community/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react'
+import { Col, Row, Input } from 'antd';
+import { SearchOutlined } from '@ant-design/icons';
+import UserCard from '../../components/Card';
+import Menu from '../../components/DropDownMenu';
```
Comment (abdullah-alsharif) [inline client/src/pages/Community/index.tsx]: merge line 4 and 5

File: client/src/pages/Community/index.tsx
Comment (abdullah-alsharif) [inline client/src/pages/Community/index.tsx:1]: spokenLanguages and practiceLanguages must be an array not string

**Review by abdullah-alsharif — APPROVED** [review]

**Review by devstm — APPROVED** [review]

**Pull request merged**